### DISCUSSION TITLE
License

### DISCUSSION THREAD

**Discussion by avanspector** [tursodatabase/turso#432] [General]
Why not retain public domain "license" from SQLite or offer dual-licensing with MIT?

**Comment by penberg** [comment]
The MIT license is the standard permissive license for open source projects.

I don't actually even like public domain very much because it creates more confusion than helps as people by default have copyright ovet the work they produce. So it's harder to prove something is public domain than licensed under an explicit license.

Of course, where Limbo borrows from SQLite we retain the original notices in place.
